packages/yarnpkg-pnpify/sources/generateSdk.ts

```diff
@@ -121,7 +121,7 @@ const TEMPLATE = (relPnpApiPath: PortablePath, module: string, {setupEnv = false
   `\n`,
   `const {existsSync} = require(\`fs\`);\n`,
   `const {createRequire, createRequireFromPath} = require(\`module\`);\n`,
-  `const {resolve} = require(\`path\`);\n`,
+  `const {resolve, dirname} = require(\`path\`);\n`,
```

Comment:
Actually, this will be imported no matter what now, could you fix that in https://github.com/yarnpkg/berry/pull/1905?

Reply:
You mean, only import the `dirname` in the case where it is required? I can do that in the morning :)

Reply:
Yeah, no rush